typings/artifacts.d.ts

```diff
@@ -121,12 +122,12 @@ declare global {
       requestSpeedIndex(data: LH.Artifacts.MetricComputationDataInput): Promise<Artifacts.LanternMetric|Artifacts.Metric>;
 
       // Lantern metrics.
-      requestLanternInteractive(data: LH.Artifacts.MetricComputationData): Promise<Artifacts.LanternMetric>;
-      requestLanternEstimatedInputLatency(data: LH.Artifacts.MetricComputationData): Promise<Artifacts.LanternMetric>;
-      requestLanternFirstContentfulPaint(data: LH.Artifacts.MetricComputationData): Promise<Artifacts.LanternMetric>;
-      requestLanternFirstCPUIdle(data: LH.Artifacts.MetricComputationData): Promise<Artifacts.LanternMetric>;
-      requestLanternFirstMeaningfulPaint(data: LH.Artifacts.MetricComputationData): Promise<Artifacts.LanternMetric>;
-      requestLanternSpeedIndex(data: LH.Artifacts.MetricComputationData): Promise<Artifacts.LanternMetric>;
+      requestLanternInteractive(data: LH.Artifacts.MetricComputationDataInput): Promise<Artifacts.LanternMetric>;
```

Comment:
@patrickhulce I believe this was your intention all along (mostly because it works, and that can't be on accident :). I typed these as `MetricComputationData` based on their calls within the other metrics where the `MetricComputationDataInput` -> `MetricComputationData` had already taken place, but they of course work without the extra artifacts. This also makes `predictive-perf.js` work above.

I had to move the optional `simulator` to `MetricComputationDataInput` as a result

Reply:
yes indeed the metric computation data is the internal one that gets augmented 👍